build/gen_stub_test.php

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env php
```

Comment:
I'm not quite sure what the purpose of this script is. Wouldn't just running `git diff` on the repository give you the same information?

Reply:
sorry, this script was quick and dirty for me. I have copy all _arginfo.h from ext/ in one folder. Also the stub.php and than i compare it. I will change it later to support simple 2 git clone folder.

Reply:
I think what Nikita meant is that you can easily verify if the refactoring was good by having a look at the `git diff` after running your version of `gen_stub.php`, in which case there shouldn't be any change in any of the `_arginfo.h` files. So this test is not really needed.

Reply:
Omg, I was so stupid.

Reply:
Don't worry :)